src/components/Errors.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+import { ErrorMessage } from '../types/ErrorMessage';
+
+type Props = {
+  errorMessage: ErrorMessage,
+  closeError: () => void,
+};
+
+export const Errors: React.FC<Props> = ({ errorMessage, closeError }) => {
+  useEffect(() => {
+    return () => {
+      setTimeout(() => {
+        closeError();
+      }, 3000);
+    }
+  }, []);
+
```

Comment:
comment from the previous mentor means that you have to clean up the `setTimeout` on unmount
```suggestion
  useEffect(() => {
    let timerId = setTimeout(() => { closeError(); }, 3000);
    
    return () => {
      clearTimeout(timerId);
    };
  }, []);

```

Reply:
thanks, I will fix it in the next task